app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -875,17 +792,15 @@ public Void execute(SFTPClient client) {
                       try {
                         for (RemoteResourceInfo info :
                             client.ls(SshClientUtils.extractRemotePathFrom(path))) {
+                          boolean isDirectory = false;
```

Comment:
I see this method (listFiles) is expired. Maybe we can remove it's usage and point to forEachChildrenFile only?

Reply:
PrepareCopyTask,CopyNode still depends on it. Not dare to remove this at this moment.